src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -254,6 +345,25 @@ def _build_key_usage(self, ext):
             decipher_only
         )
 
+    def _build_subject_alt_name(self, ext):
+        gns = self._backend._ffi.cast(
+            "GENERAL_NAMES *", self._backend._lib.X509V3_EXT_d2i(ext)
+        )
+        assert gns != self._backend._ffi.NULL
+        # GCing this should be safe. See line 964 of v3_utl.c
```

Comment:
I didn't actually want this one to be in the PR at all. That was a reference to myself. Gone now :)